polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -588,6 +588,38 @@ def update_desired_ee_pose(
 
         return update_idx
 
+
+    def start_joint_velocity(self, joint_vel_desired, Kq=None, Kqd=None, **kwargs):
```

Comment:
I would rename to `start_joint_velocity_control` to make it less confusing.
(`impedance` is less ambiguous in comparison for the other methods)